### PR TITLE
Event

### PR DESCRIPTION
**Pull request by CommanderHamster**
Mostly working, stable state to build from

### CODE REVIEW

File: app/controllers/event_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  # Edit a single event
+  def edit
+  end
+
+  # Edit Logic
+  def update
+  end
+
+  # Delete a User (probably not gonna be used)
```
Comment (kjorg50) [inline app/controllers/event_controller.rb]: in this comment, do you mean "Delete an Event"? Obviously not a big deal, just something I noticed when looking over the code

Comment (CommanderHamster) [inline app/controllers/event_controller.rb]: I think that was an accidental carry over from user, I'll fix it

File: db/seed_events.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Event.create!({:name=>"Jared and the Pussycats", :date=>"2/26/2014"})
+Event.create!({:name=>"Alex Against the Machine", :date=>"5/14/2014"})
+Event.create!({:name=>"Ben and the Boo Boo Bears", :date=>"1/26/2014"})
+Event.create!({:name=>"The Kyle Project", :date=>"7/26/2014"})
```
Comment (kjorg50) [inline db/seed_events.rb]: Just curious about the event seed data -- do they ever get associated with Venues?

Comment (CommanderHamster) [inline db/seed_events.rb]: After looking at it, it does not appear to happen. If you could fix that it would be great

**Comment by CommanderHamster** [comment]
Okay I think that I have worked out all of the immediate bugs. You still can't create from the event page, but we don't need it to be in this branch for that. Merge so that develop can be more up to date with what we have so far.

**Comment by bmbritt10** [comment]
Looks good!

**Pull request merged**